{{cookiecutter.project_name}}/.github/workflows/run-tests-fs.yml

```diff
@@ -148,7 +148,7 @@ jobs:
             ],
             "git_source": {
               "git_url": {% raw %}"${{ github.server_url }}/${{ github.repository }}",{% endraw %}
-              "git_provider": "gitHub",
+              "git_provider": "{{cookiecutter.cicd_platform}}",
```

Comment:
Can we verify diffs for the gitHub option as well (that there's no regression).

You can do two separate commits of a generated Stacks project and then diff them.